bin/emqx

```diff
@@ -163,14 +163,18 @@ relx_rem_sh() {
     # Generate a unique id used to allow multiple remsh to the same node
     # transparently
     id="remsh$(relx_gen_id)-${NAME}"
-
+    if [ -z "$WITH_EPMD" ]; then
+        EPMD_REMOTE_ARG="-start_epmd false -epmd_module ekka_epmd"
+    else
+        EPMD_REMOTE_ARG="-start_epmd true"
+    fi
     # Get the node's ticktime so that we use the same thing.
     TICKTIME="$(relx_nodetool rpcterms net_kernel get_net_ticktime)"
 
     # Setup remote shell command to control node
     exec "$BINDIR/erl" "$NAME_TYPE" "$id" -remsh "$NAME" -boot start_clean \
          -boot_var ERTS_LIB_DIR "$ERTS_LIB_DIR" \
-         -setcookie "$COOKIE" -hidden -kernel net_ticktime $TICKTIME
+         -setcookie "$COOKIE" -hidden -kernel net_ticktime $TICKTIME $EPMD_REMOTE_ARG
```

Comment:
```suggestion
         -setcookie "$COOKIE" -hidden -kernel net_ticktime $TICKTIME $EPMD_ARG
```

Reply:
Are you serious? Your suggestion will make remote_console unable to work.

Reply:
> Are you serious? Your suggestion will make remote_console unable to work.

We can work after this modification. Can you describe the failure scenario?

Reply:
Protocol 'ekka': register/listen error: eaddrinuse

Reply:
> Protocol 'ekka': register/listen error: eaddrinuse

This is because you have run `emqx start` and `emqx remote_console` on the same device. When you run `emqx start` on device A and `emqx remote_console` on device B, it works. (Need to set correct `node.name` and `node.cookie`)

Reply:
I think that after running `emqx start` on the same device, you can use `emqx attach` to enter the console

Reply:
Ok, I think I misunderstand the usage of remote_console. I think attach is so dangerous. EMQX will stop when you enter ctrl+c and we always use ctrl+c to exit.

Reply:
Yes, I also think attach is a very dangerous command, it should be used by developers rather than users. Regarding remote_console, we are also exploring best practices, and you are welcome to continue to pay attention